src/backend/common/helpers/matchstats_helper.py

```diff
@@ -0,0 +1,72 @@
+from typing import Dict, Callable, List
+
+from backend.common.consts.alliance_color import AllianceColor
+from backend.common.models.match import Match
+from backend.common.helpers.opr_helper import OPRHelper
+
+
+def get_opp_color(color: AllianceColor) -> AllianceColor:
+    return [AllianceColor.RED, AllianceColor.BLUE][color == AllianceColor.RED]
+
+
+class MatchstatsHelper:
+    EVERGREEN: Dict[str, Callable[[Match, AllianceColor], float]] = {
+        "oprs": lambda match, color: match.alliances[color]["score"],
+        "dprs": lambda match, color: match.alliances[get_opp_color(color)]["score"],
+        "ccwms": lambda match, color: (
+            match.alliances[color]["score"]
+            - match.alliances[get_opp_color(color)]["score"]
+        ),
+    }
+
+    COMPONENTS = {
+        2020: {
+            "Foul Points": lambda match, color: match.score_breakdown[color].get(
+                "foulPoints", 0
+            ),
+            "Auton div Total": lambda match, color: (

```

Comment:
this is just a placeholder stat that is just to show off custom component OPR ability

Reply:
Let's maybe use something we could publish for 2020. Like maybe cell or climb OPR?